<?xml version="1.0"?>
<launch>
  <!-- Launch the safety controller node -->
  <node pkg="final_challenge2025" exec="controller" name="safety_controller" output="screen">
    <param name="safety_topic" value="/vesc/low_level/input/safety"/>
  </node>
  <node pkg="final_challenge2025" exec="controller" name="safety_controller" output="screen">
    <param name="safety_topic" value="/vesc/low_level/input/safety"/>
  </node>
</launch>
